libraries/scripting/dependencies/src/kotlin/script/experimental/dependencies/ExternalDependenciesResolver.kt

```diff
@@ -16,7 +17,11 @@ interface ExternalDependenciesResolver {
     fun acceptsRepository(repositoryCoordinates: RepositoryCoordinates): Boolean
```

Comment:
And why there is no location here? Maybe it makes sense to add one here too.

Reply:
Including location in `addRepository` as discussed

Reply:
✅